### PR TITLE
git-review: upgrade to version 1.25.0

### PR DESCRIPTION
**Pull request by hermansc**

### CODE REVIEW

File: Library/Formula/git-review.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class GitReview < Formula
   desc "Submit git branches to gerrit for review"
   homepage "https://git.openstack.org/cgit/openstack-infra/git-review"
-  url "https://pypi.python.org/packages/source/g/git-review/git-review-1.24.tar.gz"
-  sha1 "9183b505366b842cff32132ee88d8eff44bb7c89"
+  url "https://pypi.python.org/packages/source/g/git-review/git-review-1.25.0.tar.gz"
+  sha1 "48dc683cd50d09469f621af7106dcef54af905c3"
   revision 1
```
Comment (dunn) [inline Library/Formula/git-review.rb]: Should remove this line also (resetting the revision number back to 0).

Comment (mikemcquaid) [inline Library/Formula/git-review.rb]: :+1:

**Comment by DomT4** [comment]
Please remove the revision line per @dunn's comment, and if you could bump to `sha256` at the same time that'd be incredibly appreciated. Thanks!

**Comment by DomT4** [comment]
Ping.

**Pull request closed without merging**